src/app/components/PrioBall/PrioBall.tsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+type PrioIconProps = {
+  icon: JSX.Element;
+};
+
+export default function PrioIcon({ icon }: PrioIconProps): JSX.Element {
```

Comment:
This is not an icon, it's a wrapper/container where you like to insert an icon.
What is the advantage of this component / Why shouldn't I use the icons directly?